### DISCUSSION TITLE
[20230620] General (positive) feedback

### DISCUSSION THREAD

**Discussion by kw-sudo** [rern/rAudio#1365] [General]
@rern 
@ceejayemm

The last few days I've tried a lot of things to install r/Audio 'as clean as possible' to fix or find any issues. Some have disappeared into thin air (e.g. adding NAS), some still unchanged (library view). In the end I was fine with 'i20230521 (no addons) + pacman -Syu'.

But today with the recent update + pacman -Syu, all my worries are gone. The best way is to pour the whole thing in concrete. :-D

Thx a lot
Oliver

**Comment by kw-sudo** [comment]
Sometimes I no longer understand the world. I think I need to clarify my statement from above.

I was so happy about it that I would do a clean install from the scratch. But if I choose the official recommended way (_after initial boot_) I get that 'library view error' again.

1. recent image file
2. r/Audio addons
3. pacman -Syu
4. settings

If do it 'my way' I get a sorted view including addon update is finshed (current at the above date)

1. recent image file (i20230521)
2. settings (adding NAS, library view _"album sorted by artist"_)
3. re-built library complete
4. pacman -Syu 
5. r/Audio addons (20230620)

**BUT**...as soon as I start a library update the library view error is present (no matter whether complete or only incremental). An installation only with the image file + pacman -Syu but no addons works fine.

PS: I don't know if it makes sense to reopen the old topic? Or am I really the only one with the problem?

**Reply by ceejayemm** [reply]
Why do you do a pacman update? I can't remember the last time I did this. 

Chris

**Reply by kw-sudo** [reply]
Because of _kernel update_. Okay, I'll give it a try without.

Oliver

**Reply by kw-sudo** [reply]
I've run out of ideas. Either way, the same wrong behavior.

I noticed the following: after I created library database the first time and didn't set any 'artist filters', I never scrolled to the end because the overview started with albums. But since the beginning all listed 'single artists' are 'hiding' at the end. 

This sorting issue definitely does not exist in i20230521. But I can't say from which addon update it started, because I only run them via the function button.

Oliver
